third_party/packages/ansi_up/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
+## 2.0.0-dev
```

Comment:
can you please merge in the changes from https://github.com/flutter/devtools/commit/168f4ec292d17a6f56942e5a880e25e039c54893?